utils/aliasingLightning.js

```diff
@@ -0,0 +1,100 @@
+// import I * from './item'
```

Comment:
Get rid of all of these comments - we have this data format already so it does not need to be here: https://github.com/star-ocean-anamnesiac/anamnesiac/blob/master/src/assets/data/item/weapon/arm.yml